tests/PetrolCarTests.cpp

```diff
@@ -0,0 +1,84 @@
+#include <gtest/gtest.h>
+#include "../PetrolCar.hpp"
+#include "../Gear.hpp"
+#include "../Exceptions.hpp"
+
+TEST(PetrolCarTests, 
+	ShouldHaveVelocityEqualToZeroAfterCreation)
```

Comment:
Jeśli miałbym być czepialski to nie do końca rozumiem czemu w niektórych przypadkach łamiecie linię pomimo tego, że nawet nie ma więcej niż 80/100/120 znaków i to nie jest konsekwentnie stosowane ;)

Reply:
Wynika to chyba z tego, że miałem split na dwie kolumny w edytorze - w jednej kod programu a w drugiej testy ;D